### PR TITLE
MITRE v2 Feed - Some fixes

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40563

## Description
Add TPB, Add docs to get indicator command.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ChanochShayner** [comment]
TPB task for the specific problem -
image

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md [hunk 1]
```diff
@@ -118,3 +118,35 @@ There is no context output
 | Enterprise ATT&CK | 95ecc380-afe9-11e4-9b6c-751b66dd541e |
 | PRE-ATT&CK | 062767bd-02d2-4b72-84ba-56caef0f8658 |
 | Mobile ATT&CK | 2f669986-b40b-4423-b720-4396ca6a462b |
+
+
```
Comment (yuvalbenshalom) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md:122]: please add also attack pattern documentation

**Review by yuvalbenshalom — APPROVED** [review]

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comment

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_0_5.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### MITRE ATT&CK Feed v2
+- Downgrade the Docker image to: *demisto/taxii2:1.0.0.21842* to fix an issue where the commands ***attack-pattern*** ***mitre-get-indicator-name*** does not work properly.
```
Comment (bakatzir) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_0_5.md:4]:
```suggestion
- Downgraded the Docker image to: *demisto/taxii2:1.0.0.21842* to fix an issue where the commands ***attack-pattern***, ***mitre-get-indicator-name*** did not work properly.
```

**Review by bakatzir — COMMENTED** [review]
commented on the RN

**Pull request merged**